docs/configuration/alertmanager-integration/google-managed-prometheus.rst

```diff
@@ -0,0 +1,84 @@
+Google Managed Prometheus
+==========================
+
+This guide walks you through integrating your `Google Managed Prometheus <https://cloud.google.com/stackdriver/docs/managed-prometheus>`_ with Robusta.
+
+You will need to configure two integrations: both a pull integration and a push integration.
+
+Prerequisites
+****************
+An instance of Google Managed Prometheus with the following components configured:
+
+* Prometheus Frontend (`Instructions <https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed>`_)
+* Node Exporter (`Instructions <https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/node_exporter>`_)
+* Scraping configuration for Kubelet and cAdvisor (`Instructions <https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/kubelet-cadvisor>`_)
+* Kube State Metrics (`Instructions <https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/kube_state_metrics>`_)
+
+Configure Push Integration
+********************************************
+
+To send alerts to Robusta, create an AlertManager configuration file with the name ``alertmanager.yaml``:
+
+.. code-block:: yaml
+
+   receivers:
+     - name: 'robusta'
+       webhook_configs:
+         - url: 'http://robusta-runner.default.svc.cluster.local/api/alerts'
+           send_resolved: true
+     - name: 'default-receiver'
+
+   route:
+     routes:
+       - receiver: 'robusta'
+         group_by: [ '...' ]
+         group_wait: 1s
+         group_interval: 1s
+         matchers:
+           - severity =~ ".*"
+         repeat_interval: 4h
+         continue: true
+     receiver: 'default-receiver'
+
+Apply this file as a secret to your cluster using the following command:
+
+.. code-block:: bash
+
+   kubectl create secret generic alertmanager \
+     -n gmp-public \
+     --from-file=alertmanager.yaml
+
+Verify it Works
+------------------------------
+
+Run this command to send a dummy alert to AlertManager:
```

Comment:
@pavangudiwada are you sure these are the right instructions for GMP? Did you test this?